frontend/src/pages/workspace/zoom-controller/index.tsx

```diff
@@ -3,24 +3,31 @@ import zoomOut from '@assets/icon/zoom-out.svg';
 import zoomIn from '@assets/icon/zoom-in.svg';
 import { useRecoilState } from 'recoil';
 import { zoomState } from '@context/workspace';
-
 function ZoomController() {
 	const [zoom, setZoom] = useRecoilState(zoomState);
 
 	const handleZoomOut = () => {
-		setZoom((prevZoom) => (prevZoom - 10 > 0 ? prevZoom - 10 : prevZoom));
+		setZoom((prevZoom) =>
+			prevZoom.zoom - 10 >= 50
+				? { zoom: prevZoom.zoom - 10, event: 'control' }
+				: { zoom: prevZoom.zoom, event: 'control' }
+		);
```

Comment:
```suggestion
		setZoom((prevZoom) => {
	              return { ...prevZoom, zoom: prevZoom.zoom - 10 >= 50 ? prevZoom.zoom - 10 : prevZoom.zoom };
              });
```

event는 변하지 않으니까 이렇게 spread해주고 zoom만 수정해주는게 더 명확하지 않을까요?!

Reply:
버튼을 눌렀을 때 control이고 휠을 돌렸을때 wheel로 이벤트를 바꿔줘야 정상적으로 작동해서 저렇게 짰어요. event라는 속성이 없으면 휠로 줌을 했을 때 zoomToPoint 함수가 두번 실행되는 구조가 되어서 그걸 막아주기 위해서 event라는 속성을 넣었습니다~